dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -106,6 +107,7 @@ static String getCleanedDescription(String description) {
                 description = description.substring(0, MAX_DESCRIPTION - Client.PADDING) + "...";
             }
         }
+        description = MoreObjects.firstNonNull(description, "");
```

Comment:
FYI, sorry miscommunication. 
I meant that if line 110 now was in front of line 103, you could skip the null check on like 103 altogether. 
Not worth doing a rebuild now, but for the future you can really simplify things ;)